include/usgscsm/Utilities.h

```diff
@@ -65,6 +65,15 @@ void lagrangeInterp(const int &numTime, const double *valueArray,
 double brentRoot(double lowerBound, double upperBound,
                  std::function<double(double)> func, double epsilon = 1e-10);
 
+// Use the Newton-Raphson method undistort a pixel (dx, dy), producing (ux, uy).
+void newtonRaphson(double dx, double dy, double &ux, double &uy,
+                    std::vector<double> const& opticalDistCoeffs,
+                    DistortionType distortionType, const double tolerance,
+                    std::function<void(double, double, double &, double &,
+                                       std::vector<double> const&)> distortionFunction,
+                    std::function<void(double, double, double *, 
+                                       std::vector<double> const&)> distortionJacobian);
+
```

Comment:
Somewhat of a nitpick but would it make more sense for this to be part of the `Distortion.[cpp|h]` files? I guess I don't understand why a distortion function would be put into utils. Can it be used for more than computing an undistorted pixel?

Reply:
My reasoning was that this would be a general purpose algorithm for inverting a 2D to to 2D function, to complement the current Brent algorithm which is in here as well. 

Note that Utilities.cpp already has a lot of distortion-related logic, such as computeDistortedFocalPlaneCoordinates(), getDistortionModel(), getDistortionCoeffs().

So, the function I added has general-purpose root-finding logic, but its interface is tied to lens distortion.

I can easily move it back to Distortion.cpp where it was before if it makes more sense.

Reply:
The getDistortionModel(), getDistortionCoeffs() makes some sense as other ISD get functions are also present in the Utilities files. I think this makes enough sense the `newtonRaphson` function can stay here